docs/reference/query-dsl/mlt-query.asciidoc

```diff
@@ -6,7 +6,7 @@ set of documents. In order to do so, MLT selects a set of representative terms
 of these input documents, forms a query using these terms, executes the query
 and returns the results. The user controls the input documents, how the terms
 should be selected and how the query is formed. `more_like_this` can be
-shortened to `mlt` deprecated[5.0.0,use `more_like_this` instead).
+shortened to `mlt` deprecated[5.0.0,use `more_like_this` instead].
```

Comment:
I'm not 100% sure how I wandered to this point in the docs and fixed this, but it seems like a good thing anyway.